### PR TITLE
Updated documentation, headers, support for 18-bit detectors

### PR DESCRIPTION
**Pull request by sliakat**
- Updated READMEs with current links and the latest validations with detectors using Ubuntu 22.04LTS and PICam 5.13.
- Updated headers with those from version 5.13
- Added case for readout of uint32 (18-bit detectors)
- Reworked camera open sequence to work around some oddities seen when using Pixis.

### CODE REVIEW

File: iocs/PICamIOC/iocBoot/iocPICam/st.cmd [hunk 1]
```diff
@@ -2,7 +2,7 @@ errlogInit(20000)
 
 < envPaths
 #epicsThreadSleep(20)
-dbLoadDatabase("$(TOP)/dbd/PICamApp.dbd")
+dbLoadDatabase("/home/towa/EPICS/support/areaDetector/ADPICam/iocs/PICamIOC/dbd/PICamApp.dbd")
```
Comment (MarkRivers) [inline iocs/PICamIOC/iocBoot/iocPICam/st.cmd:5]: The original line using $(TOP) is correct.  Running "make" in the iocPICam directory will produce an envPaths file that correctly defines TOP.

File: iocs/PICamIOC/iocBoot/iocPICam/st.cmd [hunk 2]
```diff
@@ -12,6 +12,8 @@ epicsEnvSet("XSIZE",  "2048")
 epicsEnvSet("YSIZE",  "2048")
 epicsEnvSet("NCHANS", "2048")
 # The search path for database files
+epicsEnvSet("ADCORE", "/home/towa/EPICS/support/areaDetector/ADCore")
+epicsEnvSet("ADPICAM","/home/towa/EPICS/support/areaDetector/ADPICam")
```
Comment (MarkRivers) [inline iocs/PICamIOC/iocBoot/iocPICam/st.cmd:16]: These additions of ADCORE and ADPICAM are not needed.  These will automatically be defined in envPaths.

File: iocs/PICamIOC/iocBoot/iocPICam/st.cmd [hunk 3]
```diff
@@ -29,7 +31,11 @@ asynSetTraceIOMask($(PORT), 0, 2)
 dbLoadRecords("$(ADCORE)/db/ADBase.template", "P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
 dbLoadRecords("$(ADPICAM)/db/PICam.template","P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
 
-# Create a standard arrays plugin, set it to get data from Driver.
+# Create a standard arrays plugin, set it to get dae_path not found.
```
Comment (MarkRivers) [inline iocs/PICamIOC/iocBoot/iocPICam/st.cmd:34]: This section appears to have garbled output from the command line ("e_path not found").  iocInit() should not be here, it appears below.

**Review by MarkRivers — CHANGES_REQUESTED** [review]
This PR looks very good except for some mistakes in st.cmd.

Those may be present because the author did not realize that they need to edit iocPICam/Makefile to set ARCH to the correct value.  It is set to windows-x64-debug on the master branch.  If running on a standard Linux system then change this to linux-x86_64.  Then run 
```
make clean
make
```
in that directory.  That will create an envPaths file that correctly defines TOP, ADCORE, and ADPICAM.

**Comment by sliakat** [comment]
Thanks for the comments, Mark. You are correct that I didn't realize the changes that needed to be made to build on Linux. I reverted st.cmd back to the original (from areaDetector/ADPICam master). Let me know if any issues!

**Comment by MarkRivers** [comment]
This looks OK to me know.  If you think it is ready I will merge it.

**Comment by sliakat** [comment]
all good on my end, Mark! Thanks again for the tip on the env generation.

**Pull request merged**